masp specs modifications (#152)
anoma/namada-docs: packages/specs/pages/masp.mdx

## Multi-asset shielded pool

The multi-asset shielded pool (MASP) is an extension to the Sapling circuit which adds support for sending arbitrary assets.

> Here we are going to showcase how it is integrated in Namada.

Reviewer: ```suggestion
This document showcases how it is integrated in Namada.
```
We want to avoid the use of first/second person
